Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM python:3.8.5-buster as base
+
+WORKDIR /todo-app
+
+RUN curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python
+ENV PATH="/root/.poetry/bin:${PATH}"
+
+EXPOSE 5000
+
+COPY pyproject.toml .
+COPY poetry.lock .
+RUN poetry config virtualenvs.create false --local
+RUN poetry install
+
+FROM base as production
+# Configure for production
+ENTRYPOINT [ "poetry" ]
+CMD ["run", "gunicorn", "--bind=0.0.0.0", "src.todolist.app:create_app()"]
+# run command: docker run --env-file .env -p 5000:8000 --mount type=bind,source="$(pwd)"/src,target=/todo-app/src todolist:prod
```

Comment:
- You shouldn't be mounting files for the production image - you should `COPY` the files over
- This command is implicitly dependent on first building the image with the tag todolist:prod